examples/metricq_get_history.py

```diff
@@ -98,9 +99,7 @@ async def aget_history(
             click.echo(aggregate)
 
 
-@click.command()
-@click.option("--server", default="amqp://localhost/")
-@click.option("--token", default="history-py-dummy")
+@metricq_command(default_token="history-py-dummy")
 @click.option("--metric", default=None)
```

Comment:
A `metric` argument or option seems to be a common occurrence. So it would make sense to me to provide a matching decorator. Ideally, this would support setting `required`. It could also check for the regex from [here](https://github.com/metricq/metricq/wiki/Metrics)